config/config.toml.example

```diff
@@ -13,10 +13,10 @@ store = "mocktikv"
 path = "/tmp/tidb"
 
 # The socket file to use for connection.
-#socket = ""
+socket = ""
```

Comment:
Why uncomment it?

Reply:
Make it consistent with other settings in the config example file.